README.md

```diff
@@ -17,9 +17,9 @@ For full dashboard functionality, upload a CSV or XLS file with the following co
 
 ***Note:** 
 - `lat` and `lon` columns are not required to utilize the dashboard, but there will be no map view if they are not included.
-- `file_urls` is not required, but there will be no sample images option if it is not included.
+- `Image_filename` and `file_url` are not required, but there will be no sample images option if it they not both included.
```

Comment:
It's a little unclear to me whether you mean to say _if neither is included_, or _if either one is not included_.

Reply:
It was supposed to be "if they are not both included", though I like the suggested "if either one is not included"